src/igvfd/schemas/biosample.json

```diff
@@ -224,31 +224,25 @@
             "type": "string",
             "format": "date"
         },
-        "related_biosamples": {
-            "title": "Related biosamples",
-            "description": "Biosamples that are related to this biosample with the relation of 'related_biosample_relation'.",
-            "comment": "A list of biosamples should all have the same relationship to this biosample.",
+        "pooled_from": {
+            "title": "Biosample(s) pooled from",
+            "description": "The biosample(s) this biosample is pooled from.",
             "type": "array",
+            "default": [],
             "uniqueItems": true,
             "items": {
-                "description": "A related biosample",
-                "comment": "See biosample.json for available identifiers.",
+                "title": "Biosample",
+                "description": "Biosample(s).",
                 "type": "string",
                 "linkTo": "Biosample"
             }
         },
-        "related_biosamples_relation": {
-            "title": "Relationship to other related biosamples",
-            "description": "The specific relationship to the biosamples in related_biosamples field",
-            "comment": "Must be submitted with related_biosamples",
+        "part_of": {
+            "title": "Part of biosample",
+            "description": "Links to a biosample which represents a larger sample from which this sample was taken regardless of whether it is a tissue taken from an organism or smaller slices of a piece of tissue or aliquots of a cell growth.",
+            "comment": "For other specific relationships such as differentiation or hosting there are separate properties.",
             "type": "string",
-            "enum": [
-                "originated_from",
-                "pooled_from",
-                "separated_from",
-                "hosted_by",
-                "derived_from"
-            ]
+            "linkTo": "Biosample"
```

Comment:
I wonder if we should make this generic to Sample instead of tied to biosample.

Reply:
I don't see the harm in making it an option if you think it is a feasible a biosample may be a part of a technical sample.

Reply:
i am leaning towards leaving it Biosample for the reason Ian brings up.